access_test.go

```diff
@@ -70,7 +70,7 @@ func TestSetupInvitedUser(t *testing.T) {
 	serverDetails := &config.ServerDetails{Url: *tests.JfrogUrl, AccessToken: *tests.JfrogAccessToken}
 	encodedCred := encodeConnectionDetails(serverDetails, t)
 	setupCmd := coreenvsetup.NewEnvSetupCommand().SetEncodedConnectionDetails(encodedCred)
-	err := setupCmd.SetupAndConfigServer()
+	_, err := setupCmd.SetupAndConfigServer()
 	assert.NoError(t, err)
```

Comment:
Let's check the also suffix, I think it is expected to be empty
```suggestion
	suffix, err := setupCmd.SetupAndConfigServer()
	assert.NoError(t, err)
	assert.Empty(t, suffix)
```